src/main/java/uk/ac/ebi/ampt2d/registry/entities/Property.java

```diff
@@ -35,8 +35,8 @@
 import java.time.ZonedDateTime;
 
 @Entity
-@EntityListeners(AuditingEntityListener.class)
-public class Property {
+@EntityListeners({AuditingEntityListener.class,EntityEventListener.class})
```

Comment:
Is there a configurable way to turn the notifications on/off? If we are sending notifications for every `write` events there could be loads during peak time.

Reply:
configured via property